creator-node/src/index.ts

```diff
@@ -111,30 +148,21 @@ const startApp = async () => {
     )
   }
 
-  await connectToDBAndRunMigrations()
-
   const nodeMode = config.get('devMode') ? 'Dev Mode' : 'Production Mode'
+
   await serviceRegistry.initServices()
```

Comment:
lets you and i both look over serviceRegistry to make sure no problems with multiple workers duplicating all the init steps
some of the steps are just bull, might make sense to move some things into the master or de-duplicate somehow

Reply:
lets talk about `_registerNodeOnL2URSM`

Reply:
made a card for what we discussed - https://linear.app/audius/issue/CON-422/make-sure-registeronursm-is-only-ran-once